cmd/bbolt/main.go

```diff
@@ -1234,13 +1232,16 @@ func (cmd *benchCommand) runWritesWithSource(db *bolt.DB, options *BenchOptions,
 				if err := b.Put(key, value); err != nil {
 					return err
 				}
+
+				results.AddCompletedOps(1)
 			}
 
+			results.ReportProgress()
+			fmt.Fprintf(cmd.Stderr, "Committing transaction...\n")
```

Comment:
The committing operation is transparent to users. How about add `fmt.Fprintf(cmd.Stderr, "Starting write iteration %d\n", i)` and `fmt.Fprintf(cmd.Stderr, "Finished write iteration %d\n", i)` at the begin and end respectively?

Reply:
Done.